README.md

```diff
@@ -67,6 +66,8 @@ let data = UserDefaults.init(suiteName:"YOUR_GROUP_ID")
 
 ### Create Widget Layout inside `android/app/src/main/res/layout`
 
+This file contains the 
+
```

Comment:
This seems like an unintentional change?
I'll be doing some adjustments to this section hopefully this week to finish up the work for documenting Jetpack Glance support so maybe remove this change?

Reply:
Ah yes, this is a mistake 😅

```suggestion
```